docs/pricing.md

```diff
@@ -1,12 +1,12 @@
 # Pricing
 
-Cirrus CI is free for Open Source projects. For private projects Cirrus CI has couple of options depending on your needs:
+Cirrus CI is free for Open Source projects. For private projects, Cirrus CI has couple of options depending on your needs:
 
 1. For private personal repositories there is a [very affordable $10 a month plan](https://github.com/marketplace/cirrus-ci/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTI=#pricing-and-setup) with 
    access to community clusters for [Linux](guide/linux.md), [Windows](guide/windows.md) and [macOS](guide/macOS.md) workloads.
-2. Buy [compute credits](#compute-credits) to access managed and pre-configured community clusters for [Linux](guide/linux.md), [FreeBSD](guide/FreeBSD.md), [Windows](guide/windows.md) and [macOS](guide/macOS.md) workloads.
-3. Configure access to [your own infrastructure](#compute-services) and [pay $10/seat/month](https://github.com/marketplace/cirrus-ci/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTM=#pricing-and-setup)
+2. Configure access to [your own infrastructure](#compute-services) and [pay a $10/seat/month](https://github.com/marketplace/cirrus-ci/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTM=#pricing-and-setup)
```

Comment:
Did you rebase with the latest changes? Pricing update in #585 changed the order here.

Reply:
Fixed.